ui/page/privacy/manual_mixer_setup_page.go

```diff
@@ -122,65 +116,110 @@ func (pg *ManualMixerSetupPage) OnNavigatedTo() {
-										return pg.mixedAccountSelector.Layout(pg.ParentWindow(), gtx)
-									})
+									return layout.Flex{Alignment: layout.Start}.Layout(gtx,
+										layout.Rigid(func(gtx C) D {
+											return pg.backFromManual.Layout(gtx)
+										}),
+										layout.Rigid(func(gtx C) D {
+											label := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleAutoOrManualG))
+											return layout.Inset{
+												Top:  values.MarginPadding10,
+												Left: values.MarginPadding5,
+											}.Layout(gtx, label.Layout)
+										}),
+										// TODO: The Label above will not go to the left unless I eat up all the space
+										// on the right here. This cannot be the correct way to implement this. Find a
+										// way to remove this spacer.
+										layout.Rigid(func(gtx C) D {
+											return layout.Spacer{Width: 100000000}.Layout(gtx)
+										}),
+									)
 								}),
 								layout.Rigid(func(gtx C) D {
-									return layout.Inset{Top: values.MarginPaddingMinus15}.Layout(gtx, func(gtx C) D {
-										return pg.mixerAccountSections(gtx, values.String(values.StrUnmixedAccount), func(gtx layout.Context) layout.Dimensions {
-											return pg.unmixedAccountSelector.Layout(pg.ParentWindow(), gtx)
-										})
-									})
+									return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+										layout.Rigid(func(gtx C) D {
+											return pg.mixerAccountSections(gtx, values.String(values.StrMixedAccount), func(gtx layout.Context) layout.Dimensions {
+												return pg.mixedAccountSelector.Layout(pg.ParentWindow(), gtx)
+											})
+										}),
+										layout.Rigid(func(gtx C) D {
+											return layout.Inset{Top: values.MarginPaddingMinus15}.Layout(gtx, func(gtx C) D {
+												return pg.mixerAccountSections(gtx, values.String(values.StrUnmixedAccount), func(gtx layout.Context) layout.Dimensions {
+													return pg.unmixedAccountSelector.Layout(pg.ParentWindow(), gtx)
+												})
+											})
+										}),
+										layout.Rigid(func(gtx C) D {
+											return layout.Spacer{Height: 10}.Layout(gtx)
+										}),
+										layout.Rigid(func(gtx C) D {
+											card := pg.Theme.Card()
+											card.Color = pg.Theme.Color.Gray4
+											return card.Layout(gtx, func(gtx C) D {
+												gtx.Constraints.Min.X = gtx.Constraints.Max.X
+												gtx.Constraints.Min.Y = 110
+												return layout.Inset{Top: values.MarginPadding10, Left: values.MarginPadding16,
+													Right: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
+													return layout.Inset{Top: values.MarginPaddingMinus15, Left: values.MarginPaddingMinus15,
+														Right: values.MarginPaddingMinus15, Bottom: values.MarginPaddingMinus30}.Layout(gtx, func(gtx C) D {
+														return layout.Flex{Alignment: layout.Start}.Layout(gtx,
+															layout.Flexed(1, func(gtx C) D {
+																gtx.Constraints.Max.X = 40
```

Comment:
Is it wrong to use raw numbers here? Doing this in a few places.

Reply:
you can have something like `values.MarginPadding40`

Reply:
I think I changed them all to use `values`